src/audio/src/src.c

```diff
@@ -763,7 +763,7 @@ static int src_copy(struct comp_dev *dev)
 	ret = src_get_copy_limits(cd, source, sink);
 	if (ret) {
 		trace_src_error("No data to process.");
-		return ret;
+		return 0;
```

Comment:
Best to make the trace message a verbose level warning (since this is non fatal).

Reply:
ok